group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupTest.java

```diff
@@ -1046,4 +1052,107 @@ public void testIsInStatesCaseInsensitive() {
         assertTrue(group.isInStates(Collections.singleton("stable"), 1));
         assertFalse(group.isInStates(Collections.singleton("empty"), 1));
     }
+
+    @Test
+    public void testSupportsProtocols() {
+        ConsumerGroup consumerGroup = createConsumerGroup("foo");
+        ConsumerGroupMemberMetadataValue.ClassicProtocolCollection collection1 =
```

Comment:
collection1 and collection2 are a bit hard to follow. Should we rather use `rangeProtocol` and `roundRobinAndRangeProtocols` or something along those lines?